manifest.json

```diff
@@ -32,7 +32,10 @@
       }
     },
     {
-      "name": "vtex.graphql-server:resolve-graphql"
+      "name": "vtex.storefront-permissions:resolve-graphql"
+    },
+    {
+      "name": "vtex.b2b-organizations-graphql:resolve-graphql"
```

Comment:
I don't understand these policies very well, but did you get an error that required changing these? Just asking cause graphql-server is the one that actually resolves graphql, I find it weird to see policies for resolving graphql related to these other apps.

Reply:
I didn't find the documentation, but when I tested and split the clients to pass the provider app name correctly, the request didn't work, and I got error 403 in the log, related to the IAM resource. So I replaced the `graphql-server` policy for each app.

Reply:
Got it, this new code is calling the apps directly instead of graphql-server, so it needs different policies.